tsup.config.ts

```diff
@@ -35,9 +41,17 @@ const config: Options = {
   // shown.
   silent: true,
 
+  // Generate sourcemaps as separate files.
+  // https://tsup.egoist.dev/#generate-sourcemap-file
+  sourcemap: true,
+
   // Split the output into chunks. This is useful for tree-shaking.
   // https://tsup.egoist.dev/#code-splitting
   splitting: true,
+
+  // The environments to target when generating the bundles.
+  // https://tsup.egoist.dev/#target-environment
+  target: ['es2020'],
```

Comment:
Do we want to be explicit about this? It seems that this is correctly inferred from the tsconfig.